src/main/java/uk/gov/hmcts/ccd/v2/external/resource/CaseDocumentResource.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.ccd.v2.external.resource;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.springframework.hateoas.RepresentationModel;
+import uk.gov.hmcts.ccd.v2.external.controller.CaseDocumentController;
+import uk.gov.hmcts.ccd.v2.external.domain.CaseDocumentMetadata;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class CaseDocumentResource extends RepresentationModel {
+
+    @JsonProperty("documentMetadata")
+    private CaseDocumentMetadata documentMetadata;
+
+    public CaseDocumentResource(@NonNull String caseId, @NonNull String documentId, CaseDocumentMetadata documentMetadata) {
```

Comment:
make sure we are returning the v1 indicator of the resource in the api responses, in the Content-Type headers.

Reply:
New resource is added with following value :         public static final String CASE_DOCUMENT = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-document.v1+json;charset=UTF-8";